pkg/scheduler/objects/queue_test.go

```diff
@@ -1166,8 +1169,7 @@ func TestQueueProps(t *testing.T) {
 	leaf, err = createDynamicQueue(parent, "leaf", false)
 	assert.NilError(t, err, "failed to create leaf queue")
 	assert.Assert(t, leaf.isLeaf && !leaf.isManaged, "leaf queue is not marked as unmanaged leaf")
-	assert.Equal(t, len(leaf.properties), 1, "leaf queue properties size incorrect")
-	assert.Equal(t, leaf.properties[configs.ApplicationSortPolicy], "stateaware", "leaf queue property value not as expected")
+	assert.Equal(t, len(leaf.properties), 0, "leaf queue properties size incorrect")
```

Comment:
The fact that this test breaks means we have regressed somewhere. This test should not break by default unless we specifically want it to break.